### DISCUSSION TITLE
Shopify App backed by Pocketbase

### DISCUSSION THREAD

**Discussion by pnmcosta** [pocketbase/pocketbase#3555] [Show and tell]
Hi,

We're on the last steps of launching https://www.socialclique.app/ a Shopify App built with a React frontend and Pocketbase as a backend.

From the moment we picked Pocketbase I couldn't have been happier, especially the level of assistance, updates and guidance from @ganigeorgiev we've got since - you sir, are a legend 🥇 - many thanks for everything you do for this project, and indirectly to all of ours. 🙇

The stack is ready, ~~what we've been struggling with the most is Meta (which we need to integrate with Instagram). We submitted our business verification on September 22nd, but it's been crickets since then. 🦗

If anyone got the Meta magic touch or know someone on the inside who could nudge things along? Please do let me know, happy to reach out outside GitHub.~~ and we just got verified! 🥳 Let the fun begin.

All the best,
Pedro Costa

**Comment by ohbob** [comment]
I'm curious, how you will manage the login brute-force? as limiting requests is not the part of pb ?

**Reply by pnmcosta** [reply]
Good question, on the admin side of things eventually we might restrict access to certain ips/networks, and we plan to roll our own lite admin that will probably use a dedicated auth provider, we'll phase out the use of PB admins eventually. 

On the front end we don't need to massively worry about it as all logins are oauth based (both Shopify's and IG users), hopefully their infra will cover us a bit, I've purposely disabled email/password auth for our user collection on PB.

**Reply by ganigeorgiev** [reply]
_Just a side note - due to the bcrypt passwordHash functions being slow, inherently this acts also as a very rudimentary brute-force "protection" for the login/register routes (eg. ~1-2 requests per second; [see auth with password bechmark](https://github.com/pocketbase/benchmarks/blob/master/results/hetzner_cax11.md#user-auth-with-password-expected-to-be-slow-due-to-passwordhash-verification)). Captcha support and one-time-codes via email are planned in the future but that will be somewhere after the logs refactoring._
